### ISSUE TITLE
Users entering a game without scenes displayed error message and cannot see each other

### ISSUE THREAD

**Issue by DAckDesign** [Gerdofal/foundryvtt-cyberpunkred#80]
Users trying to connect to a hosted lobby that doesn't have a scene would see the `User USERNAME lacks permission to update the Wold setting cyberpunkred.systemMigrationVersion` error. The users cannot be seen by the game master nor by each other, and they cannot see the game master either.

**Comment by Gerdofal** [comment]
When you say they cannot be seen, you mean they don't show up on the logged in user list at all?

**Comment by DAckDesign** [comment]
Yes exactly, the logged in user list only displays the user being used, despite technically not being the only one connected to the lobby.

**Comment by Gerdofal** [comment]
I was unable to duplicate this in testing. Is it possible this is related to any other module? CPRED system doesn't do anything with scenes.

Were there any errors in the console? (Press F12 and look at console tab)
